### ISSUE TITLE
Mathmatical Characters Are Being Deleted

### ISSUE THREAD

**Issue by jj49er** [clach04/pebble_patch_tts#1]
The script is deleting mathematical characters in addition to punctuation. This renders calculator apps still useless. Can certain mathematical characters be excluded in this script, or in an alternate script for certain apps requiring those characters, but not being able to process the punctuation?

**Comment by clach04** [comment]
Bug or feature :smiley_cat: ? For my use case, I'm not interested in that (I personally want any words) BUT that's a great idea for those that need it, and for specific apps :+1: 

Ideas:

  * ifdef in the code to preserve certain character, then use the cflags option in rockgarden, the control via flags into the python script
  * offer different python scripts (all together), patch with math, patch without.

To implement this, you'd need to use a custom C function (or hack isgraph, which I would recommend avoiding as it would be non-obvious to maintainers versus a custom written function). So yes totally doable, but you need to mask those specific characters or re-implement the filter function.

Long term, I think the best bet would be for when Cobble https://github.com/pebble-dev/mobile-app arrives to have per pebble app settings on the phone for controlling speech to text processing. I've been thinking about this for a while but not been on the Discord for a while to float the idea. I did wonder about updating the dictation server but it has no idea which app is running where as the phone app does (it only knows which user).

Long story short, I'll happily review a well written PR :-)

**Comment by jj49er** [comment]
I apologize if my issue/request was not placed properly. I wasn't sure
where/how to post it. I'm not versed in GitHub etiquette.

 I don't know C. I looked at the code and got a basic idea of what it was
excluding, but I don't know how to limit the exclusions. I thought it might
be a simple tweek, but it doesn't sound like it is.

I don't check the Discord very often either. Is the new app really that
close? The last time I looked it still seemed very far away.

Again, thank you for what you've done! The calculator issue is very mild in
comparison to getting Snowy working again.
